.github/workflows/r-cmd-check.yaml

```diff
@@ -8,9 +8,11 @@ on:
 jobs:
   check-package:
     strategy:
+      fail-fast: false
```

Comment:
you might want to revert this as it makes the CI less wasteful (the whole matrix fails if one entry fails).

Reply:
I probably copied this blindly from somewhere.

Reply:
No I added it for testing so I could see what worked and what didn't.

Reply:
I prefer to have it on to avoid waste but you do get more information if its off.